src/cpu/x64/brgemm/brgemm_containers.hpp

```diff
@@ -89,6 +88,10 @@ struct brgemm_kernel_container_t {
     std::set<std::shared_ptr<brgemm_kernel_t>,
             decltype(brgemm_kernel_container_t::brgemm_kernel_cmp) *>
             set_;
```

Comment:
Hi, I'm not sure about this. As I understood in case of we do not use global storage every container has their own set. I think this code do not lead to static initialization issues, so I left it as is